tests/e2e/utils/flinkStatement.ts

```diff
@@ -76,7 +75,7 @@ export async function submitFlinkStatement(
  */
 export async function stopStatement(webview: FrameLocator) {
   // Check if the stop button is disabled
-  const stopButton = webview.getByTestId(FlinkStatementTestIds.stopStatementButton);
```

Comment:
No need for an enum at this point, future branch could improve these by using a Flink statement result webview page object model (https://github.com/confluentinc/vscode/issues/2159)